src/main/resources/interface-specification.yml

```
@@ -3875,6 +3875,64 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /import/eservices/{tenantId}/presignedUrl:
```

tenantId should be taken from the jwt, if we leave the caller to pass the value we lose the authorization check
```suggestion
  /import/eservices/presignedUrl:
```